user-guide/Troubleshooting/RemoteLogCollection.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: RemoteLogCollection
+---
+
+# Remote Log Collection
+
+## What is it?
+
+The Remote Log Collection feature will allow our experts to instantly gather data from all nodes in your system. The ultimate goal is to enable our them to gather the necessary information from your system without your intervention. With this functionality experts from Skyline can trigger the LogCollector tool remotely, so they can collect the logs and memory dumps they need for their investigation.
+
+Currently the packages generated by the LogCollector are being saved on the DataMiner System, but in a future increment these packages will automatically be transferred to a secure storage in the cloud. By doing this the experts can immediately access the packages without any intervention from our users.
```

Comment:
I feel like there should also be a comma after "By doing this".
It also sounds weird to talk about "our users" when the guide is for the users themselves, maybe change it to "a user"?